extra-dev/packages/coq-tactician-stdlib/coq-tactician-stdlib.8.16.dev/opam

```diff
@@ -0,0 +1,62 @@
+opam-version: "2.0"
+name: "coq-tactician-stdlib"
+synopsis: "Recompiles Coq's standard libary with Tactician's instrumentation loaded"
+description: """
+  *** WARNING *** This package will overwrite Coq's standard library files.
+
+  This package recompiles Coq's standard library with Tactician's (`coq-tactician`)
+  instrumentation loaded such that Tactician can learn from the library. When you
+  install this package, the current `.vo` files of the standard library are backed
+  in the folder `user-contrib/Tactician/stdlib-backup`. The new `.vo` files are
+  equivalent to the originals, except that they also contain Tactician's tactic
+  databases. After installation of this package, all other Coq developments that
+  are installed will also need to be recompiled. The 'tactician recompile' command
+  line utility can help with this.
+  Upon removal of this package, the original files will be placed back.
+"""
+homepage: "https://coq-tactician.github.io"
+dev-repo: "git+https://github.com/coq-tactician/coq-tactician-stdlib"
+bug-reports: "https://github.com/coq-tactician/coq-tactician-stdlib/issues"
+maintainer: "Lasse Blaauwbroek <lasse@blaauwbroek.eu>"
+authors: "Lasse Blaauwbroek <lasse@blaauwbroek.eu"
+messages: [
+  "*** WARNING ***"
+  "This package will overwrite Coq's standard library files."
+  "A backup of the original files will be placed under Coq's"
+  "library directory at user-contrib/tactician-stdlib-backup/"
+  "and they will be restored when you remove this package."
+  "After installation of this package, all other Coq packages"
+  "also need to be recompiled. Running the 'tactician recompile'"
+  "command-line utility will help with this process."
+]
+post-messages: ["
+--- The standard library was successfully recompiled ---
+
+In order to finish the process, you should run
+
+tactician recompile
+" {success}]
+depends: [
+  "coq" {>= "8.16" & < "8.17~"}
+  "coq-tactician"
```

Comment:
@LasseBlaauwbroek are you sure this package should work with any `coq-tactician` version? Shouldn't the following make more sense?
```suggestion
  "coq-tactician" {= version}
```

Reply:
Well, there is no restriction on one particular version of Tactician. As long as it is compatible with Coq 8.16 (which is ensured by the version constraint on the `coq-tactician` package. I'm not sure what the best method for that is, but `= version` seems to restrictive to me.